cardano-testnet/src/Testnet/Types.hs

```diff
@@ -148,6 +150,18 @@ isTestnetNodeSpo = isJust . poolKeys
 nodeSocketPath :: TestnetNode -> SocketPath
 nodeSocketPath = File . H.sprocketSystemName . nodeSprocket
 
+-- | Connection data for the first node in the testnet
+node0ConnectionInfo :: MonadTest m => TestnetRuntime -> m LocalNodeConnectInfo
```

Comment:
What about exposing an index parameter?

Reply:
I figured that for `>0` indices, the connection info won't be that useful - those are relay nodes. I can add an index.